crepecake/src/main/java/com/google/cloud/tools/crepecake/http/Request.java

```diff
@@ -38,6 +38,12 @@ public BlobHttpContent getHttpContent() {
     return body;
   }
 
+  // TODO: Add to ConnectionTest.
+  public Request setAuthorization(Authorization authorization) {
+    headers.setAuthorization(authorization.asList());
```

Comment:
The HttpHeaders setAuthorization method isn't very well documented, I'm afraid. Hard to tell what's happening.

I do suggest converting to a list here rather than in Authorization.